internal/test/factory/vote.go

```diff
@@ -2,12 +2,29 @@ package factory
 
 import (
 	"context"
+	"testing"
 	"time"
 
+	"github.com/stretchr/testify/require"
 	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
 	"github.com/tendermint/tendermint/types"
 )
 
+// ErrorHandler checks an error and returns true if the error is
+// non-nil. Pass an ErrorHandler to a function in a case where you
+// might want to use a *testing.T but cannot because one is not
+// available (e.g. as is the case in some e2e test fixtures and in
+// TestMain.)
+type ErrorHandler func(error) bool
```

Comment:
I don't think anything checks the return value of this. Can we drop the `bool` entirely and have this be a simple error logger?

Reply:
oh I thought I deleted this entirely

Reply:
I was going to suggest that, but it's still being used.